### PR TITLE
Bug fix for latest SDK integration

### PR DESCRIPTION
**Pull request by ba2512005**
To address dependency and reference issues for windows 10 python 3.7

### CODE REVIEW

File: object_detection.py [hunk 1]
```diff
@@ -20,8 +20,9 @@
 # Imports the Anki Vector SDK
 import anki_vector
 from anki_vector.util import degrees
+import anki_vector.camera
 
-robot = anki_vector.Robot(anki_vector.util.parse_command_args().serial, enable_camera_feed=True)
+robot = anki_vector.Robot(anki_vector.util.parse_command_args().serial)#, enable_camera_feed=True)
```
Comment (unbug) [inline object_detection.py:25]: ` enable_camera_feed=True` defualt is `True` in the latest SDK?

File: object_detection.py [hunk 2]
```diff
@@ -120,13 +120,13 @@ def show_image(file_name):
...
     print('Say {}'.format(text))
-    robot.say_text(text)
+    robot.behavior.say_text(text)
 
 
 def analyze():
     stand_by()
     show_camera()
-    robot_say('My lord, I found something interesting. Give me 5 seconds.')
+    robot_say('Wall-E, I found something interesting. Give me 5 seconds.')
```
Comment (unbug) [inline object_detection.py]: Can we not update this? Because I have to update the docs and the video :(

File: object_detection.py (hunk 1 above)
Comment (ba2512005) [reply object_detection.py:25]: It seems to work with that line commented out. it was giving the error listed in the bugs section when it was run with it in there with the new SDK. I tried it out, it works.

File: object_detection.py (hunk 2 above)
Comment (ba2512005) [reply object_detection.py]: Sure, I just wanted to test changing the script

**Review by ba2512005 — COMMENTED** [review]
Requested changes made

File: object_detection.py (hunk 1 above)
Comment (unbug) [reply object_detection.py:25]: Can you see the video streaming shows on Vector's screen?

Comment (ape007) [reply object_detection.py:25]: robot = anki_vector.Robot(anki_vector.util.parse_command_args().serial, enable_camera_feed=True)
TypeError: __init__() got an unexpected keyword argument 'enable_camera_feed'

**Pull request merged**